evm/src/test/scala/pravda/evm/translate/opcode/TranslateTests.scala

```diff
@@ -33,13 +34,15 @@ object TranslateTests extends TestSuite {
       }
     }
 
-    'Contracts - {
+    /*  'Contracts - {
```

Comment:
???

Reply:
that's why we should rewrite them to `plaintest`, leave it for now, but it's an important thing we should implement before expanding our tests